src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -179,6 +176,17 @@ StaticLLMPipeline::StaticLLMPipeline(
 ) : StaticLLMPipeline(path, path.string(), device, config) {
 }
 
+void StaticLLMPipeline::start_chat(const std::string& system_message) {
+    if (system_message.empty()) {
+        m_history.push_back({{"role", "system"}, {"content", system_message}});
+    }
+    m_is_chat_conversation = true;
+};
+void StaticLLMPipeline::finish_chat() {
```

Comment:
```suggestion

void StaticLLMPipeline::finish_chat() {
```

Reply:
What is the difference?

Reply:
just formatting, for readability a blank line between function definitions